sonic-xcvrd/xcvrd/xcvrd_utilities/port_event_helper.py

```diff
@@ -226,7 +239,13 @@ def _handle_port_add(self, port_change_event):
         if port_change_event.port_index not in self.physical_to_logical:
             self.physical_to_logical[port_change_event.port_index] = [port_name]
         else:
+            # Append the new logical port and sort the list
             self.physical_to_logical[port_change_event.port_index].append(port_name)
+
+            self.physical_to_logical[port_change_event.port_index] = natsorted(
+                self.physical_to_logical[port_change_event.port_index], key=lambda x: x.lower()
+            )
+
```

Comment:
@mihirpat1 do you think we can use `from sortedcontainers import SortedSet` instead of doing sort every time?
`